tests/test-pool-search.rb

```diff
@@ -165,7 +165,46 @@
   #
   # test that a date range includes an expected item
   #
-  #
+  it "#{url} should return one or more items" do
+    post search_endpoint, { :query => "author:{jefferson}", :pagination => { :start => 0, :rows => 1 }  }
+    expect_status( 200 )
+
+    expect( json_body[:group_list].count ).to be > 0
+
+    # extract the first publication date from the results
+    first_pub_date = Helpers.pool_results_first_pub_date( json_body )
+
+    # extract the first title from the results
+    first_title = Helpers.pool_results_first_title( json_body )
```

Comment:
Since titles can be the same across multiple records with potentially different dates, this should test based on the id.